src/cli/cli.cc

```diff
@@ -1246,6 +1263,27 @@ int main (const int argc, const char* argv[]) {
           settings["android_allow_cleartext"] = "";
         }
       }
+
+      if (android_enable_standard_ndk_build) {
+        settings["android_defaultConfig_externalNativeBuild"].assign(
+          "    externalNativeBuild {\n"
+          "      ndkBuild {\n"
+          "        arguments \"NDK_APPLICATION_MK:=src/main/jni/Application.mk\"\n"
+          "      }\n"
+          "    }"
+        );
+
+        settings["android_android_externalNativeBuild"].assign(
+          "  externalNativeBuild {\n"
```

Comment:
these key seems awkward and potentially error prone, maybe something like:

```
android_external_native_build
```

Reply:
@jwerle happy to change this, was trying to stay with the formula of having everything start with android_ that is in the template, but I think I got a bit meta.

Reply:
no worries! just wanting things to be consistent 🌴